dask/array/fft.py

```diff
@@ -86,19 +64,64 @@ def _ihfft_out_chunks(a, n, axis):
     return chunks
 
 
-fft = _fft_wrap(npfft.fft, np.complex_, _fft_out_chunks)
+out_chunk_fns = {'fft': _fft_out_chunks,
+                 'ifft': _fft_out_chunks,
+                 'rfft': _rfft_out_chunks,
+                 'irfft': _irfft_out_chunks,
+                 'hfft': _hfft_out_chunks,
+                 'ihfft': _ihfft_out_chunks}
+
 
+def fft_wrap(fft_func, kind=None, dtype=None):
+    """ Wrap 1D complex FFT functions
 
-ifft = _fft_wrap(npfft.ifft, np.complex_, _fft_out_chunks)
+    Takes a function that behaves like ``numpy.fft`` functions and
+    a specified kind to match it to that are named after the functions
+    in the ``numpy.fft`` API.
 
+    Supported kinds include:
 
-rfft = _fft_wrap(npfft.rfft, np.complex_, _rfft_out_chunks)
+        * fft
+        * ifft
+        * rfft
+        * irfft
+        * hfft
+        * ihfft
+
+    Examples
+    --------
+    >>> parallel_fft = fft_wrap(np.fft.fft)
+    >>> parallel_ifft = fft_wrap(np.fft.ifft)
+    """
+    if kind is None:
+        kind = fft_func.__name__
```

Comment:
Indeed. Playing around with it locally seems ok. Maybe we should test a couple of `n`s for good measure.